fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt

```diff
@@ -0,0 +1,85 @@
+package com.microsoft.fluentui.tokenized.navigation
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.compositionLocalOf
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.ControlTokens
+import com.microsoft.fluentui.theme.token.controlTokens.*
+import com.microsoft.fluentui.tokenized.tabItem.TabItem
+
+
+data class TabData(
+    var title: String,
+    var icon: ImageVector,
+    var selectedIcon: ImageVector = icon,
```

Comment:
Woul it help if we used the selected and filled icons as we had before here?

Reply:
what Icon to be shown on TabItem depend on what index has been passed to TabBar. Hence, both the icon need to be store in data class.